code/game/objects/items/weapons/twohanded.dm

```diff
@@ -373,7 +373,8 @@
 		if(!isliving(target))
 			return
 		else
-			target.Weaken(4)
+			if(!istype(src, /obj/item/weapon/twohanded/chainsaw/rubber))

```

Comment:
In this case, it you would likely want to define a toy subtype of two-handed items. That said, don't do what I suggested, because there's probably some things in the twohanded attack proc or something further up the object tree that you want to call.